CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.6.2 (2019-10-01)
+
+- Fixed datepicker input directive to not convert an invalid input value into a Date. [#76](https://github.com/blackbaud/skyux-datetime/pull/76)
```

Comment:
Should this say `...into a Date object" ?

Reply:
If it's an object, I think Alex is right. ... Also, we are missing `the` after `Fixed`. ... Also, it always sounds awkward to me when we say we made a change `to not [verb here]`. I'd suggest changing this to `Fixed the datepicker input directive to no longer convert invalid input values into Date objects.`